src/gwt/src/org/rstudio/core/client/files/FileSystemItem.java

```diff
@@ -288,6 +303,15 @@ public final int compareTo(FileSystemItem other)
 
    }
 
+   private final static String normalizeSeparatorsNative(String path)
```

Comment:
Shouldn't this be called `normalizeSeparators()` or `normalizeSeparatorsPosix()`? The name implies you are using native separators here (ie, using `\` on Windows) but the behavior here is the opposite.

Reply:
Yes, you're right, I can rename this